concordium-sdk/src/main/java/com/concordium/sdk/responses/nodeinfo/NodeInfo.java

```diff
@@ -19,7 +19,7 @@
 @ToString
 public class NodeInfo {
 
-    private static final ZoneId UTC_ZONE = ZoneId.of("UTC");
+    public static final ZoneId UTC_ZONE = ZoneId.of("UTC");
```

Comment:
This should probably be moved into a separate file now it's used more places.

Reply:
moved to a separate constants file.